benchmark/vm_iclass_super.yml

```diff
@@ -0,0 +1,20 @@
+prelude: |
+  class C
+    def m
+      1
+    end
+
+    ("A".."M").each do |module_name|
+      eval <<-EOM
+          module #{module_name}
+            def m; super; end
+          end
+          prepend #{module_name}
+      EOM
+    end
+  end
+
+  obj = C.new
+benchmark:
+  vm_super: obj.m
```

Comment:
@eileencodes can you fix the benchmark name please?  I think we did locally, but it doesn't look pushed. 🙇🏻‍♂️